tests/stormpy_interface_test.py

```diff
@@ -0,0 +1,183 @@
+# This test includes integration tests for the interfaces.stormpy module,
+# using the example from the EECI 2020 computer lab.
+# Details of these tests can be found at
+# the Stormpy computer session of EECI 2020:
+#
+# https://www.cds.caltech.edu/~murray/wiki/index.php?title=EECI_2020:_Computer_Session:_Stormpy
+#
+# All the tests here are run only if stormpy is available.
+#
+# * light_analysis_test() corresponds to Example 1: traffic light (analysis)
+# * compose_test() corresponds to Example 2: traffic light + vehicle (analysis)
+# * synthesis_test() corresponds to Example 4: policy synthesis
+
+import os
+import unittest
+from tulip.transys.compositions import synchronous_parallel, apply_policy
+
+# Set up paths to all the models
+model_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "models")
+light_path = os.path.join(model_path, "light.pm")
+ma_path = os.path.join(model_path, "ma.nm")
+mh_path = os.path.join(model_path, "mh.pm")
+out_model_path = os.path.join(model_path, "tmpout.nm")
+
+skip_test = False
+try:
```

Comment:
It is best for tests collected here to fail if `stormpy` is not installed, rather than automatically skip if `ImportError` is raised.

Because we use `./run_tests.py` to drive collection and performance of tests, the act of testing involves a decision about which tests to run. As such, we assume the user wants to perform a test if it is included by `./run_tests.py`, so failure should occur if dependencies are missing.

To be clear, the pattern here is a valid for organizing tests, but it is not consistent with the current pattern of Tulip testing.

Reply:
For example, like [tests/lily_test.py](https://github.com/tulip-control/tulip-control/blob/9b9a743d1486ded5a91eba2fbbb4536559b85f5f/tests/lily_test.py) and [tests/gr1pyint_test.py](https://github.com/tulip-control/tulip-control/blob/9b9a743d1486ded5a91eba2fbbb4536559b85f5f/tests/gr1pyint_test.py)

Reply:
I see. Thanks. I'll fix this.

Reply:
Fixed in 6f868e1.

I also added new tests in this PR (except stormpy_interface_test) to run_test.py in 3291787.

Reply:
It seems like even though I don't add stormpy_interface_test to run_tests, it's still run in travis. Do you know how to disable it?

Reply:
Actually I want it to run on Travis. After I find a way to satisfy the `stormpy` dependency in Travis CI builds, then the tests will start to pass.